### PR TITLE
Fix monitoring

### LINKED ISSUE #7: Datadog monitoring integration
Monitoring integration needs fixing.
- It's currently broken.
- It's design to ping every 5 seconds, _from every web worker_, which is unnecessary (should be a separate process for now).
- It doesn't have any test coverage.

### PR DESCRIPTION
**Pull request by mezis**
Closes #7.

Mainly fixes broken queue length metric delivery (to Datadog or logs), but fixing how non-web processes/dynos get loaded became necessary, so I revamped that.

### CODE REVIEW

**Comment by tompave-roo** [comment]
LGTM

File: routemaster/services/scheduler.rb [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-  scheduler = Rufus::Scheduler.new
-
-  scheduler.every '5s' do
-    tags = [
-      "env:#{ENV['RACK_ENV']}",
-      'app:routemaster'
-    ]
-
-    Routemaster::Models::Subscription.each do |subscription|
-      Routemaster::Services::DeliverMetric.deliver(
```
Comment (tompave-roo) [inline routemaster/services/scheduler.rb:18]: `deliver` could not be called on the module, I guess that this was the broken part, right?

Comment (mezis) [inline routemaster/services/scheduler.rb:18]: Part of it. The thing wouldn't actually load at all, either. I refrained from a `git blame` during all this, for fear `@mezis` would show up :)

**Comment by martyphee** [comment]
LGTM

**Pull request merged**